packages/admin-panel/src/VizBuilderApp/App.jsx

```diff
@@ -56,8 +31,12 @@ export const App = ({ Footer, homeLink, logo }) => {
     <StateProvider>
       <SimplePageLayout logo={logo} homeLink={homeLink}>
         <Container>
-          {/** Workaround for handling issues with this nested app */}
-          {renderMatches(matches)}
+          <Routes>
```

Comment:
this wasn't directly related but just cleaning up this work around because it turns out it's not necessary